tests/std/tests/P0896R4_views_take_while_death/test.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <ranges>
+#include <span>
+
+#include <test_death.hpp>
+using namespace std;
+
+[[maybe_unused]] constexpr auto lambda = [x = 42](int) { return x == 42; };
+using DWV                              = decltype(ranges::take_while_view{span<int, 0>{}, lambda});

```

Comment:
Copy-paste relic: `DWV` was a `drop_while_view`, but this is a `take_while_view`.

Reply:
Yeah, I wanted to rename them all to R, as it is really difficult to keep al those 2-3 letter acronyms around when comparing the different tests